README.md

```diff
@@ -1,4 +1,5 @@
-# Muter 
+![Muter logo](Docs/Images/Muter%20Logo.svg)
```

Comment:
This pretends not to work in the diff view, but I believe it will work once GitHub generates a staticusercontent URL? Maybe?

Reply:
Ah, it works if you view the file directly on this branch. Rich diff view is just broken. https://github.com/muter-mutation-testing/muter/blob/513480750286b63c9f144275ac48b585cef08499/README.md